osbenchmark/workload/params.py

```diff
@@ -1086,15 +1120,27 @@ def _update_request_params(self):
             self.PARAMS_NAME_ALLOW_PARTIAL_RESULTS, "false")
         self.query_params.update({self.PARAMS_NAME_REQUEST_PARAMS: request_params})
 
+    def _get_query_neighbors(self):
+        if self.query_type == self.KNN_QUERY_TYPE:
+            return Context.NEIGHBORS
+        if self.query_type == self.MIN_SCORE_QUERY_TYPE:
+            return Context.MIN_SCORE_NEIGHBORS
+        if self.query_type == self.MAX_DISTANCE_QUERY_TYPE:
+            return Context.MAX_DISTANCE_NEIGHBORS
```

Comment:
Using `elif` will be more appropriate.

Reply:
Ummm, I used to use `elif`, @VijayanB and I agreed to updated to current condition check for cleaner read. It has validation step before this

Reply:
The effect is the same in this case, but using `elif` makes it more readable, especially since the cases are mutually exclusive.  I'll leave it to you.